tests/compile-fail/overflow_check_conditional.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#![deny(overflow_check_conditional)]
+
+fn main() {
+	let a: u32 = 1;
+	let b: u32 = 2;
+	if a + b < a {
+
+	}
```

Comment:
No, but you should add it in the test file to ensure it does not. (the test suite checks that only line with a `//~ERROR` trigger errors, so just add something with `a + b < c` with no such comment)